src/main/java/example/App.java

```diff
@@ -7,11 +7,13 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
 
 /**
  * Example app using elide-spring.
  */
 @SpringBootApplication
+@EntityScan
```

Comment:
I think we covered this - but I don't remember.  Why do we need EntityScan again?